internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -882,12 +894,32 @@ func flattenSqlVirtualMachineTempDbSettings(input *sqlvirtualmachine.SQLTempDbSe
 	}
 	attrs := make(map[string]interface{})
 
-	if input.Luns != nil {
-		attrs["luns"] = *input.Luns
+	if input.luns != nil {
```

Comment:
Same here, field name of [SQLTempDbSettings](https://github.com/Azure/azure-sdk-for-go/blob/f4fc5f70ccfd445da5c860fb0e24fd96afa38119/services/preview/sqlvirtualmachine/mgmt/2021-11-01-preview/sqlvirtualmachine/models.go#L1527). in Terraform the attribute name is `luns` and in the sdk struct its `Luns`

Reply:
Ok this should be resolved